doc/installation/livemedia.rst

```diff
@@ -47,6 +47,7 @@ Notes
 
 * To install Guest Additions, select **Devices** > **Insert Guest Additions CD image...**  (top left of the VirtualBox Window). Then, open a terminal (Ctrl+Alt+t), go to "/media/nest/VBOXADDITIONS.../" and run "sudo bash VboxLinuxAdditions.run".
 
-* To set the correct language layout for your keyboard (e.g., from "US" to "DE"), you can use the program "lxkeymap", which you start by typing "lxkeymap" in the terminal.
+* To set the correct language layout for your keyboard (e.g., from "US" to "DE"), open a terminal and type: "sudo dpkg-reconfigure keyboard-configuration". Reboot now.
```

Comment:
"Reboot now" sounds like an order. What about "After setting the correct keyboard map you have to reboot the system to activate the changes"?

Reply:
I followed your suggestion.